docs/data/charts/radar/DemoRadarSeriesHighlight.tsx

```diff
@@ -63,18 +78,21 @@ export default function DemoRadarSeriesHighlight() {
 const series = [
   {
     id: 'usa',
+    type: 'radar' as const,
```

Comment:
Would it make sense to add `type: 'radar'` to `withOptions` instead?

Same applies for the other demos that share this code.